packages/sfui/frameworks/vue/components/VsfSlider/VsfSlider.vue

```diff
@@ -64,6 +64,11 @@ const [container, slider] = useSlider(
 );
 const onClickPrev = () => slider.value?.prev();
 const onClickNext = () => slider.value?.next();
+
+const focusHandler = (event: FocusEvent) => {
+  const elementIndex = Array.prototype.indexOf.call(slider.value?.container.children, event.target);
```

Comment:
```suggestion
  const elementIndex = Array.from(slider.value?.container.children).findIndex(el => el === event.target);
```
I think using `Array.from` and `findIndex` is more readable IMHO, what do you think?

Reply:
Yes, that's better, thx.